index.md

```diff
@@ -8,6 +8,469 @@ This is some welcome text.
+method. If you are getting this error, please check the indentation levels in 
+your code carefully.
+
+```python
+def add(lst):
+    """ Calculates the sum of the elements in the given list.
+    
+    @type lst: list
+    @rtype: bool
+    """
+    temp = 0
+    for item in lst:
+        temp += item
+        
+return False # Error on this line
+```
+
+### E0107 Nonexistent operator {#E0107}
+
+### E0108 Duplicate argument name {E0108}
+
+This error occurs if there are duplicate argument names in function 
+definitions. If you are getting this error, please make sure all the arguments 
+have distinct names.
+
+```python
+def add(lst, lst):
+    """
+    @type lst: list
+    @type lst: list
+    @rtype: int
+    """
+    temp = 0
+    for item in lst:
+        temp += item
+    return temp
+```
+
+### E0110 Abstract class instantiated {#E0110}
+
+### W0101 Unreachable {#W0101}
+
+This error occurs when there is some code behind a "return" or "raise" 
+statement. This code will never be accessed, therefore check your code 
+carefully if you are getting this error. 
+
+```python
+def add(lst):
+    """
+    @type lst: list
+    @rtype: int
+    """
+    temp = 0
+    for item in lst:
+        temp += item
+    return temp
+    temp += 1 # Error on this line
+```
+
+### W0102 Dangerous default value {#W0102}
+
+This error occurs when a mutable value such as a list or dictionary is given a 
+default value in the function or method definition.
+
+```python
+def add(lst=[]):
+    """ Calculates the sum of the elements in the given list.
+
+    @type lst: list
+    @rtype: int
+    """
+    temp = 0
+    for item in lst:
+        temp += item
+    return temp
+```
+
+### W0104 Pointless statement {#W0104}
+
+This error occurs when a statement doesn't have any effect. This may not be 
+what was intended, so please check your code carefully. 
+
+```python
+def add(lst):
+    """
+    @type lst: list
+    @rtype: int
+    """
+    temp = 0
+    for item in lst:
+        temp += item
+    temp # Error on this line
+```
+
+### W0105 Pointless string statement {#W0105}
+
+This error occurs when string statement has no effect. This may not be 
+what was intended, so please check your code carefully. This error is a special 
+case of the error W0104.
+
+```python
+def letters(lst):
+    """
+    @type lst: list
+    @rtype: int
+    """
+    temp = ""
+    for letter in lst:
+        temp += letter + " "
+    "end" # Error on this line
+```
+
+### W0106 Expression not assigned {#W0107}
+
+This error occurs when an expression that is not a function call is not 
+assigned to anything. If you are getting this error, please check your code 
+carefully.
+ 
+ ```python
+ def add(lst):
+    """
+    @type lst: list
+    @rtype: int
+    """
+    temp = 0
+    for item in lst:
+        temp += item
+    5 + 2 # Error on this line
+```
+
+### W0107 Unnecessary pass {#W0107}
+
+This error occurs when a "pass" statement is used that can be avoided (or has 
```

Comment:
Is it ok to just state that we can check by removing `pass` and executing it again, and if there is no difference it means that `pass` is unnecessary? Or would you want me to explain that the `pass` in the example has no effect as it is in the end? I'm unable to find words to explain it better